FILESTRUCTURE.md

```diff
@@ -0,0 +1,67 @@
+# Vue File Structure
+
+You'll find our Vue files in `components` and `pages`. The files are structured in the order: template, script, and style. One thing to note; we want to keep any and all styling in `<style>`. This means that we try not to use Tailwind's classes directly in a file's `<template>`. Rather, we make a class name, and in the file's `<style>`, call `@apply` on top of any extra styling.
+
+## Special Directories
+
+You can create the following extra directories, some of which have special behaviors. Only `pages` is required; you can delete them if you don't want to use their functionality.
+
+### `assets`
+
+The assets directory contains your uncompiled assets such as Stylus or Sass files, images, or fonts.
+
+More information about the usage of this directory in [the documentation](https://nuxtjs.org/docs/2.x/directory-structure/assets).
+
+### `components`
+
+The components directory contains your Vue.js components. Components make up the different parts of your page and can be reused and imported into your pages, layouts and even other components.
+
+More information about the usage of this directory in [the documentation](https://nuxtjs.org/docs/2.x/directory-structure/components).
+
+### `composables`
+
+The composables directory contains your [Vue.js composables](https://vuejs.org/guide/reusability/composables.html).
+
+More information about the usage of this directory in [the documentation](https://v3.nuxtjs.org/guide/directory-structure/composables).
+
+### `i18n`
+
+`i18n` (short for internationalizaion), is where we'll be storing our locales for translating this site into different languages.
```

Comment:
Typo, and we may as well use present tense, too. Pretty sure we'll miss updating this page/not care to do so later on.

```suggestion
`i18n` (short for internationalization), is where we store our locales for translating this site into different languages.
```